recipes-bsp/device-tree/files/uz7ev-evcc/system-bsp.dtsi

```diff
@@ -178,9 +180,40 @@
 
 };
 
+&psgtr {
+	/* pcie, sata, usb3, dp */
+	clocks = <&gtr_clk0>, <&gtr_clk1>, <&gtr_clk2>, <&gtr_clk3>;
+	clock-names = "ref0", "ref1", "ref2", "ref3";
+};
+
+/* ULPI SMSC USB3320 */
+&usb0 {
+	status = "okay";
+};
+
+&dwc3_0 {
+   status = "okay"; 
+/*	dr_mode = "otg"; */
+/*	dr_mode = "peripheral"; */
+	dr_mode = "host";
+/*	maximum-speed = "super-speed"; */
+/*	snps,usb3_lpm_capable; */
+/* snps,enable_auto_retry; */
+   phy-names = "usb3-phy"; 
+	/* <psgtr_phandle> <lane_number> <controller_type> <instance> <refclk> */
+	phys = <&psgtr 2 PHY_TYPE_USB3 0 2>;
+};
+
 &sata {
 	status = "okay";
 	phy-names = "sata-phy";
-	phys = <&lane1 PHY_TYPE_SATA 1 1 125000000>;
+	/* <psgtr_phandle> <lane_number> <controller_type> <instance> <refclk> */
+	phys = <&psgtr 1 PHY_TYPE_SATA 1 1>;
 };
 
+// Just in case the user ever wants to disable the DisplayPort output
```

Comment:
I don't think this is necessary
I think the code should stay as clean and clear as possible. 
If the user wants to disable the DP he can ask for support or look it up on the internet. We can't start pre-coding every functionalities the end user may or may not want.